docs/notebooks/pre_executed/wrapping_redback.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    print(sig)  # (time, mass_ejecta, velocity_ejecta, opacity
</pre><p>Should we have a redshift term between time and mass_ejecta?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lightcurvelynx/pull/622/discussion/#comment-2561022641'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Yes. I just removed the comment though as it isn't adding anything